appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy

```diff
@@ -58,31 +64,36 @@ class UserPhotoTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
-	def 'User updates user photo; the previous photo is still retrievable'()
+	def 'Richard updates his photo; it is still retrievable but not linked to him'()
```

Comment:
I think I suggested this name, but have one more request: please change it into
``Richard updates his photo; the old one is still retrievable but not linked to him``